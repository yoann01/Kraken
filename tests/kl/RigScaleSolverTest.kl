
require Kraken;

operator entry(){
	RigScaleSolver solver();

    Boolean debug = true;
    Scalar rigScale = 3.5;

    Mat44 target();

    solver.solve(debug, rigScale, target);

    report(Xfo(target).sc);
}
