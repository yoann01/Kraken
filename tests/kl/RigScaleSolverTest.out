{x:+3.5,y:+3.5,z:+3.5}